Source/WebCore/page/scrolling/mac/ScrollingTreeScrollingNodeDelegateMac.h

```diff
@@ -76,8 +82,7 @@ class ScrollingTreeScrollingNodeDelegateMac final : public ThreadedScrollingTree
 
     void releaseReferencesToScrollerImpsOnTheMainThread();
 
-    RetainPtr<NSScrollerImp> m_verticalScrollerImp;
-    RetainPtr<NSScrollerImp> m_horizontalScrollerImp;
+    std::unique_ptr<ScrollerPairMac> m_scrollerPair;
```

Comment:
Does it have to be a std::unique_ptr or can we just store by value?

Reply:
Changed to store by value.